upup/pkg/fi/cloudup/alitasks/securitygrouprule.go

```diff
@@ -89,7 +82,7 @@ func (s *SecurityGroupRule) Find(c *fi.Context) (*SecurityGroupRule, error) {
 	actual := &SecurityGroupRule{}
 	// Find securityGroupRule with specified ipProtocol, securityGroupId,SourceGroupId
 	for _, securityGroupRule := range describeResponse.Permissions.Permission {
-		if !equalsIgnoreCase(string(securityGroupRule.IpProtocol), fi.StringValue(s.IpProtocol)) {
+		if !strings.EqualFold(string(securityGroupRule.IpProtocol), fi.StringValue(s.IpProtocol)) {
```

Comment:
I have replace this func in  https://github.com/kubernetes/kops/pull/8118

Reply:
Aha, will remove it then.

Reply:
Done